app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,7 @@
     package="com.worldsnas.starplayer">
 
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.ACCESS_MEDIA_LOCATION"/>
```

Comment:
android o majarahash

Reply:
android va mokafat, bedoune in permission roye emulator javab midad